taskt/Core/Automation/Commands/Data Commands/ModifyStringCommand.cs

```diff
@@ -1,61 +1,61 @@
 ﻿using System;
 using System.Collections.Generic;
+using System.Text;
 using System.Windows.Forms;
 using System.Xml.Serialization;
+using taskt.Core.Automation.Attributes.ClassAttributes;
+using taskt.Core.Automation.Attributes.PropertyAttributes;
+using taskt.Core.Utilities.CommonUtilities;
 using taskt.UI.CustomControls;
 using taskt.UI.Forms;
-using taskt.Core.Utilities.CommonUtilities;
 
 namespace taskt.Core.Automation.Commands
 {
     [Serializable]
-    [Attributes.ClassAttributes.Group("Data Commands")]
-    [Attributes.ClassAttributes.Description("This command allows you to modify a string")]
-    [Attributes.ClassAttributes.UsesDescription("Use this command when you want to modify text")]
-    [Attributes.ClassAttributes.ImplementationDescription("This command uses the String Functions to Modify Text.")]
+    [Group("Data Commands")]
+    [Description("This command performs a specified operation on a string to modify it.")]
     public class ModifyStringCommand : ScriptCommand
     {
         [XmlAttribute]
-        [Attributes.PropertyAttributes.PropertyDescription("Please select a variable or text to modify")]
-        [Attributes.PropertyAttributes.InputSpecification("Select or provide a variable from the variable list")]
-        [Attributes.PropertyAttributes.SampleUsage("**vSomeVariable**")]
-        [Attributes.PropertyAttributes.Remarks("")]
-        public string v_userVariableName { get; set; }
+        [PropertyDescription("Text Data")]
+        [InputSpecification("Provide a variable or text value.")]
+        [SampleUsage("A sample text || {vStringVariable}")]
+        [Remarks("Input data other than string (text) type will cause error.")]
```

Comment:
Change to "Providing data of a type other than a 'String' will result in an error."

Reply:
done